src/App.module.css

```diff
@@ -59,7 +59,11 @@
 }
 
 .options {
-  display: flex;
+  /* display: flex; */
   margin-bottom: 1rem;
-  justify-content: space-between;
+  /* justify-content: space-between;  */
```

Comment:
Same here.

Reply:
@claeusdev I just commented out the styles so if you didn't want the changes I made, I could just easily use the original styles, but I have removed them :)